### PR TITLE
Update list of packages for Cloud 7

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

File: suse-openstack-cloud-rpm-list [hunk 1]
```diff
@@ -76,36 +89,50 @@ openstack-heat-doc
 openstack-heat-engine
 openstack-heat-plugin-heat_docker
 openstack-heat-templates
+openstack-heat-test
```
Comment (vuntz) [inline suse-openstack-cloud-rpm-list]: I'm surprised we ship that...

Comment (rsalevsky) [inline suse-openstack-cloud-rpm-list]: It's in the tempest category as `<package name="openstack-heat-test" supportstatus="unsupported"/>`.

Comment (rsalevsky) [inline suse-openstack-cloud-rpm-list]: @vuntz Should I move this to the testing media?

Comment (vuntz) [inline suse-openstack-cloud-rpm-list]: @dirkmueller any idea?

Comment (dirkmueller) [inline suse-openstack-cloud-rpm-list]: this is for the heat tempest plugin. I need to double check if we need it.

**Comment by vuntz** [comment]
@rsalevsky I think you should just merge and update the package for now. We can always have a followup PR in case we remove the test package...

**Pull request merged**